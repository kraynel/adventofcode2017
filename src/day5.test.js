const { getNumberOfSteps } = require("./day5");
const fs = require("fs");

describe("getNumberOfSteps", () => {
  it("should check if is valid 1", () => {
    expect(
      getNumberOfSteps(`0
3
0
1
-3`)
    ).toBe(5);
  });

  it("should log the correct value", done => {
    fs.readFile("inputs/day5.txt", "utf8", (err, data) => {
      expect(getNumberOfSteps(data)).toMatchSnapshot();
      done();
    });
  });
});
