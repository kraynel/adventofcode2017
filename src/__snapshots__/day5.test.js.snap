// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getNumberOfSteps should log the correct value 1`] = `355965`;
